### PR TITLE
Making comment

### PR DESCRIPTION
**Pull request by tanguyguillo**
Integration of commrnets on page detail and adding author

### CODE REVIEW

File: app/controllers/postsController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Application\Controllers\PostsController;
+
+use Application\Controllers\Controller;
+use Application\Models\PostsModel\PostsRepository;
+use Application\Core\Database\Database\DatabaseConnection;
+
+// to use requite here have to fbe fix.... still a issue
+require_once(ROOT . '/app/core/database/database.php');  // used for $connection
```
Comment (castroCrea) [inline app/controllers/postsController.php:10]: You might need to do a general `require` of all files at the index.php

**Review by castroCrea — APPROVED** [review]

File: app/controllers/postsController.php (hunk 1 above)
Comment (tanguyguillo) [reply app/controllers/postsController.php]: It's what i did: i put all the requires in my config.php file which is loaded in the index.php file and it's ok

Comment (tanguyguillo) [reply app/controllers/postsController.php]: .... but a can make a special file for it also....

**Comment by castroCrea** [comment]
en 2 tu peux merge ça aussi

**Comment by tanguyguillo** [comment]
I re did it

[image: Capture d’écran 2022-07-11 à 11.11.23.png]




Le dim. 10 juil. 2022 à 22:39, castroCrea ***@***.***> a
écrit :

File: app/controllers/postsController.php (hunk 1 above)
Comment (tanguyguillo) [reply app/controllers/postsController.php]: I resolve this issue by coding a special  autoload

`function my_autoloader($class)
{
    $delim = '\\';
    $myArray = explode($delim, $class);
    $newarray = array_slice($myArray, 1, -1);
    $delim = "/";
    $class2 = implode($delim, $newarray);
    $path = (ROOT . "/app/" . $class2 . ".php");

    if (file_exists($path)) {
        require $path;
    }
}
spl_autoload_register('my_autoloader');
`

**Pull request merged**